docs/connect.rst

```diff
@@ -0,0 +1,52 @@
+.. _schemaregistry_kafka_connect:
+
+Kafka Connect
+=============
+
+Kafka Connect and Schema Registry integrate to capture schema information from connectors. :ref:`Kafka Connect converters <connect_converters>`
+provide a mechanism for converting data from the internal data types use by Kafka Connect to data types represented as Avro.
+The AvroConverter will automatically register schemas generated by Source connectors. Sink Connectors will receive schema
```

Comment:
Sink Connectors -> Sink connectors (all other mentions to connectors in the text is lower case)